src/services/docker/docker.ts

```diff
@@ -23,7 +23,8 @@ export function getImageName(uri: string): string {
 }
 
 export async function getImageDigest(uri: string): Promise<string> {
-  const imageName = getImageName(uri);
+  // String docker.io/ from any image name, as they get removed in the `docker images` list
```

Comment:
Looks like getImageName is not longer needed. Could you remove it?

Reply:
Yeah.  I left it in because it could be useful for other stuff, but you're right might as well remove

Reply:
@agathanatasha removed!